### ISSUE TITLE
[console] Follow Drupal version number for Console

### ISSUE THREAD

**Issue by kgaut** [hechoendrupal/drupal-console#2156]
Drupal 8.1 is there, the correct console version for it is 1.0
Drupal 8.0 is still out there, even if it's deprecated, we know how it works ;)
Drupal 8.2 will be released in few months, what will be the correct console version for it ? 1.1 ?

In order to keep things simple I'm suggesting that Console versions should follow the Drupal version they are made for :
- Bump current master (1.0) to 8.1.0
- Bump 0.11 to 8.0.0
- Create a 8.2.x branch when work will start for Drupal 8.2.0

**Comment by darol100** [comment]
**tl;dr** No too many relationship to make this happening.

@kgaut, Drupal Core minor release are base on adding new funcionality/ improvements to Drupal Core. On the another hand, Drupal Console release are issue improvements/ new funcionality, which are not necessary have any relationship with Drupal Core. 

> Drupal 8.2 will be released in few months, what will be the correct console version for it ? 1.1 ?

We can't predict the minor version because again, we do release base on current improvements of the Console and not Drupal Core. And most of this improvements are community driven, meaning that is whenever we have time to build it. Compare to Drupal Core, which has some core maintainers that get paid to contribute to Drupal Core.

Most likely we will have a version that supports 8.2 but predicting a minor version is very hard. 

> Bump current master (1.0) to 8.1.0

This will mean that we have a stable version of Drupal Console, and we cant change anything from the API. BTW we are on alpha (1.0-alpha), which means we are going to change a lot of major things from the Drupal Console core before the stable 1.0 . 

> Create an 8.2.x branch when work will start for Drupal 8.2.0

We will be creating a version DC x2, however, does not have any relationship with what is happening in Drupal Core. This will depend on what API's we need to break the API from version 1.0.

The only reason why in Drupal.org projects add the major version as part of the release is because they want the users to understand which module have compatibility release with current major Drupal Core. However, a minor release does not any relationship the current major release.


Here are my two cents, I do not think we should do this......

@jmolivas, @enzolutions , @omero  what is your opinion on this ?

**Comment by LOBsTerr** [comment]
@jmolivas it looks like we still don't use two years after. So, I we can close it

**Comment by jmolivas** [comment]
Agree on @darol100 no need to since we increase minor version based on project features, not drupal releases.
